contrib/python-pyqt6-sip/template.py

```diff
@@ -0,0 +1,24 @@
+pkgname = "python-pyqt6-sip"
```

Comment:
should probably be python-pyqt6_sip

Reply:
that's so darned ugly

but yeah, changed